Makefile

```diff
@@ -46,7 +46,7 @@ endif
 
 # Timeout shortly before the 600 second travis silence timeout
 # (method=thread to support xdist)
-PYTEST_OPTS := -v --timeout=550 --timeout_method=thread
+PYTEST_OPTS := -v --timeout=550 --timeout_method=thread -p no:logging
```

Comment:
What does this do?

Reply:
That instructs `pytest` not to print logs on failure. This was causing duplicate output on failure because it was showing both the captured stdout and stderr, and calls to `logging.info` and `logging.debug`. Since we are writing logging output to stdout anyway, that'd duplicate things. This should make the output a bit more readable :wink: